Front-End/src/PageM.js

```diff
@@ -52,585 +53,571 @@ function SignIn(props) {
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                        </tr>
-                        <tr>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                        </tr>
-                    </table>
+        const fetchData = async () => {
+            try {
+                const response = await fetch(url);
+                const json = await response.json();
+                console.log(json);
+                setAdvice(json);
+                setIsLoaded(true);
+            } catch (error) {
+                console.log("error", error);
+            }
+        };
 
+        fetchData();
+    }, []);
+    if (isLoaded) {
+        return (
+            <Page selected={props.selected} id="IntakeDataTable">
+                <div>
+                    <br></br>
+
+                    <p className="selection-label" id="Intake-Data-Table">Intake Data Tables</p>
+                    <div className="textArea">
+                        <table border="1" width="100%">
+                            <tbody>
+                                <tr>
+                                    <th>Mechanism</th>
+                                    <th>Number Of Parts</th>
+                                    <th>Size Of Part</th>
+                                    <th>Type Of Stock</th>
+
+                                    <th> Thickness (Extrusion)</th>
+                                    <th> Thickness (Plate)</th>
+                                    <th> Material (Plate)</th>
+                                    <th>  Length in Inches</th>
+                                    <th>  Part Number</th>
+                                    <th> Date</th>
+                                </tr>
+                                <tr>
+                                    <td>{advice[0][0]}</td>
+                                    <td>{advice[0][1]}</td>
+                                    <td>{advice[0][2]}</td>
+                                    <td>{advice[0][3]}</td>
+                                    <td>{advice[0][4]}</td>
+                                    <td>{advice[0][5]}</td>
+                                    <td>{advice[0][6]}</td>
+                                    <td>{advice[0][7]}</td>
+                                    <td>{advice[0][8]}</td>
+                                    <td>{advice[0][9]}</td>
+                                </tr>
```

Comment:
Same as above, we won't need this `<tr>` where we manually set `{advice[0][*]}` since now we create a `<tr>` for every row in the database

Reply:
fixed.